### ISSUE TITLE
TypeError: aws_sdk_1.QLDBSession is not a constructor

### ISSUE THREAD

**Issue by vihuy** [awslabs/amazon-qldb-driver-nodejs#89]
<img width="883" alt="Screen Shot 2021-05-14 at 4 05 39 PM" src="https://user-images.githubusercontent.com/26733094/118248305-5bb84e00-b4ce-11eb-88f2-10d994e18fce.png">

**Comment by justing-bq** [comment]
We will look into this and get back to you shortly!

**Comment by byronlin13** [comment]
Hi @vihuy,

Thanks for opening this issue!

What version of the driver are you using?

**Comment by byronlin13** [comment]
Hi @vihuy ,

I was able to reproduce your issue by setting the AWS SDK dependency version to <[2.525](https://github.com/aws/aws-sdk-js/blob/master/CHANGELOG.md#25250) (This version was when QLDBSession was introduced).

Please update your AWS-SDK dependency to [v2.841.0](https://github.com/awslabs/amazon-qldb-driver-nodejs/blob/v2.2.0/package.json#L40) as specified by the driver's package.json.

**Comment by DavidBM** [comment]
Still happening with the 2.910.0

**Comment by vihuy** [comment]
> Hi @vihuy,
> 
> Thanks for opening this issue!
> 
> Could you provide us with more information:
> 
> * What is your driver version? 
> * What is your AWS SDK version?
> * What is your Node.js version?
> * Is it possible for you to provide steps to repro?

Answer: 
> * What is your driver version?  2.1.1
> * What is your AWS SDK version? 2.841.0
> * What is your Node.js version? v11.13.0
> * Is it possible for you to provide steps to repro? 

I use react for my app

I installed it according to the document and it gives me this error.

**Comment by DavidBM** [comment]
I solved the problem. For us the problem is that QLDBSession seems to only exist in a Nodejs context. We were using in our cypress tests and they run in electron, which doesn't have the same Nodejs APIs. 

Once we started using tasks to run aws-sdk code in nodejs, all worked well.

It will be the same for environments like the browser, electron, phonegap, react native, etc. It only seems to run in nodejs.
